ecpy/utils/widgets/qt_list_str_widget.py

```diff
@@ -99,9 +99,21 @@ def on_selection(self):
                        for index in widget.selectedIndexes()]
             if indexes:
                 if self.multiselect:
+                    # HINT force an access so that the new change (which is
+                    # swallowed by enaml) so that the notifications are
+                    # properly propagated
+                    self.selected_indexes
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Statement seems to have no effect](https://www.codacy.com/app/marul/ecpy/file/2642045851/issues/source?bid=3033427&fileBranchId=3321358#l105)